webapp/locales/en.json

```diff
@@ -30,7 +30,7 @@
         "unavailable": "Unfortunately, public registration of user accounts is not available right now on this server.",
         "title": "Join Human Connection!",
         "form": {
-          "description": "To get started, enter your e-mail address:",
+          "description": "To get started, you can register here for free:",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-07T10:44:12Z" title="Tuesday, January 7th 2020, 11:44:12 am +01:00">Jan 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -30,7 +30,7 @@
         "unavailable": "Unfortunately, public registration of user accounts is not available right now on this server.",
         "title": "Join Human Connection!",
         "form": {
-          "description": "To get started, enter your e-mail address:",
+          "description": "To get started, you can register here for free:",
```


I'm ok with the English @Mogge